example/burrow_integration/get*Transactions.py

```diff
@@ -0,0 +1,269 @@
+import os
```

Comment:
Please, rename the file, You are using forbidden character in Windows:
https://stackoverflow.com/a/31976060/1350091

Reply:
I suggest name: `get_transactions.py`